src/api/general-services/pipeline-manage/constructors/create-new-job-if-not-exist.js

```diff
@@ -58,7 +60,7 @@ const createNewJobIfNotExist = (context, step) => {
           },
         },
         spec: {
-          releaseName: `pipeline-${experimentId}`,
+          releaseName,
```

Comment:
I don't understand much here but shouldn't this be releaseName: releasename, ?

Reply:
@ogibson-biomage  That's a good intuition, but in es6 (aka modern javascript) a short hand notation is supported when the attribute name matches the name of an exiting variable https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#new_notations_in_ecmascript_2015

We are even enforcing a linter rule that would undo your suggested change: https://eslint.org/docs/rules/object-shorthand